python/cudf/cudf/core/index.py

```diff
@@ -243,6 +249,12 @@ def _values(self):
         else:
             return column.column_empty(0, masked=False, dtype=self.dtype)
 
+    def _clean_nulls_from_index(self):
```

Comment:
I was hoping this method wouldn't be needed for `RangeIndex`, but I guess `DataFrame` and `Series` are currently relying on it existing for all Index types. That being the case, can you add a `NotImplemented` version in the base index class?